### DISCUSSION TITLE
How to satisfy the policy "Require a merge strategy"

### DISCUSSION THREAD

**Discussion by xtomik** [cribeiro84/azure-devops-pull-request-hub#264] [Q&A]
Hello, 

this azure devops extension is awesome and very hepful.  Thank you.

But how can i satisfy this policy "Require a merge strategy"? It's always red.

![obrazek](https://github.com/cribeiro84/azure-devops-pull-request-hub/assets/5384752/d55b5e49-c385-46f4-956e-a7eb2b75ecab)

We use Squash merge only policy
![obrazek](https://github.com/cribeiro84/azure-devops-pull-request-hub/assets/5384752/55c0b234-985d-44b7-8434-7929111d15dc)

**Comment by cribeiro84** [comment]
Hey @xtomik, thanks for opening this discussion. This item is related to a branch policy set on your repo or default repo configuration that relates to which merge type is enforced: stash, fast-forward, rebase, so on.. it will depended on how your configuration is done. Hope this helps! 😃
